infrastructure/adminservices-test/altinn-monitor-test-rg/k6_tests_rg_rbac.tf

```diff
@@ -0,0 +1,87 @@
+resource "kubernetes_cluster_role_v1" "dev_access" {
+  metadata {
+    name = "dev-access"
+  }
+
+  rule {
+    api_groups = [""]
+    resources  = ["configmaps"]
+    verbs      = ["get", "list", "watch", "delete"]
+  }
+  rule {
+    api_groups = [""]
+    resources  = ["pods"]
+    verbs      = ["get", "list", "watch"]
+  }
+  rule {
+    api_groups = ["k6.io"]
+    resources  = ["testruns"]
+    verbs      = ["get", "list", "watch", "create", "update", "patch", "delete"]
+  }
+  rule {
```

Comment:
This one is a bit hard to get right: https://kubernetes.io/docs/concepts/security/rbac-good-practices/#listing-secrets

For now I think it's fine that developers can get access to their own secrets. If we see a need to have secrets that can't be read by developers we will probably need to implement something that allows them to validate that their secrets exist in the cluster without giving them access to the contents of the secret.

Reply:
Yeah, and if we don't have it today we should wait to handle the "problem" until the use case is there